scripts/create_gitlab_snapshot.py

```diff
@@ -22,21 +25,34 @@
 from azul.logging import (
     configure_script_logging,
 )
+from azul.types import (
+    CompositeJSON,
+    JSON,
+)
 
 log = logging.getLogger(__name__)
 
 
-def main():
+def main(argv):
+    import argparse
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument('--no-restart',
+                        default=False,
+                        action='store_true',
+                        help='Leave the EC2 instance in a stopped state after'
+                             ' the snapshot has been created.')
+    args = parser.parse_args(argv)
     require(config.terraform_component == 'gitlab',
             "Select the 'gitlab' component ('dev.gitlab' or 'prod.gitlab', for example).")
     volume = gitlab_volume_info()
-    attachments = volume['Attachments']
-    if attachments:
-        instance = one(attachments)
+    instance: CompositeJSON = only(volume['Attachments'])
```

Comment:
CompositeJSON is for values than can be either a list or a dict. This value is either `None` or `JSON`.